tests/math/test_functions.py

```diff
@@ -1851,12 +1851,17 @@ def test_block_diag(tensors):
 
 class TestBlockDiagDiffability:
 
-    expected = lambda self, x, y: np.array(
+    expected = lambda self, x, y: (

```

Comment:
how come this changed?

Reply:
This was implemented such that the output shape of `expected` matched the Autograd behaviour, and the other interfaces had to modify this shape/access the expected value "strangely" to make up for the difference in behaviour.
Now the output of `expected` is simply the (now uniform) output shape of the Jacobian :)